### ISSUE TITLE
Test for WCAG technique G142: Using a technology that has commonly-available user agents that support zoom

### ISSUE THREAD

**Issue by hannolans** [quailjs/quail#221]
http://www.w3.org/WAI/GL/2013/WD-WCAG20-TECHS-20130711/G142

The objective of this technique is to ensure content can be scaled uniformly by using a Web technology supported by user agents that change text size via a Zoom tool.

Content authored in technologies that are supported by user agents that can scale content uniformly (that is, zoom into content) satisfy this Success Criterion. Because this technique relies completely on user agent functionality, it is critical to test with a wide variety of user agents.

This technique requires that the zoom function preserve all spatial relationships on the page and that all functionality continues to be available.
## Procedure
1. Display content in a user agent
2. Zoom content to 200%
3. Check whether all content and functionality is available
## QUAIL

This test for this technique should ALWAYS return a pass, as this is completely a user agent issue. We only implement this for statistics and coverage. Test 3 is covered by failure F69 (http://www.w3.org/WAI/GL/2013/WD-WCAG20-TECHS-20130711/F69)
